daily-challenges/2024-10/21/min241021.spec.ts

```
@@ -0,0 +1,58 @@
+  - Đặt vé: Hỏa lực 1,2,3,4
+  - Verify giỏ hàng có 1 sản phẩm
+  - Đi tới trang giỏ hàng
+  - Verify tổng tiền đúng với giá đã thêm ngoài trang chủ
+*/
+import { test, expect } from '@playwright/test'
+let username = 'vn84'
+let pwd = 'StrongPassword@123'
+let quantity = '1'
+
+test.describe('Authentication Tests', () => {
+  test.beforeEach(async ({ page }) => {
```

Biome

[error] This let declares a variable that is only assigned once.

'pwd' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)